sources/oceania/au/nsw/Maxar-1920-Bushfires-PreEvent.geojson

```diff
@@ -0,0 +1,134 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "Maxar_1920_Bushfires_PreEvent",
+        "name": "Maxar 1920 Bushfires Pre-event",
+        "type": "tms",
+        "category": "photo",
+        "permission_osm": "explicit",
+        "license": "CC-BY-NC-4.0",
+        "license_url": "https://www.digitalglobe.com/ecosystem/open-data/australia-wildfires",
+        "attribution": {
+            "url": "https://www.digitalglobe.com/ecosystem/open-data/australia-wildfires",
+            "text": "©2020 DigitalGlobe",
+            "required": true
+        },
+        "url": "https://cogeo.xyz/tiles/6b6f479fbacd9a42e9e38bc5c0c6889f009beae13b07742ec4a1648f/{zoom}/{x}/{y}.png",
+        "max_zoom": 19,
+        "min_zoom": 0,
```

Comment:
@grischard the `min_zoom` for this layer is zoom 11 see: https://cogeo.xyz/tiles/6b6f479fbacd9a42e9e38bc5c0c6889f009beae13b07742ec4a1648f/tilejson.json

Reply:
Fixed in 92fa1ed, thank you!